charts/consul/templates/crd-httproutetimeoutfilters.yaml

```diff
@@ -0,0 +1,118 @@
+{{- if .Values.connectInject.enabled }}
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.9.2
+  creationTimestamp: null
+  name: httproutetimeoutfilters.consul.hashicorp.com
+  labels:
+    app: {{ template "consul.name" . }}
+    chart: {{ template "consul.chart" . }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+    component: crd
+spec:
+  group: consul.hashicorp.com
+  names:
+    kind: HTTPRouteTimeoutFilter
+    listKind: HTTPRouteTimeoutFilterList
+    plural: httproutetimeoutfilters
+    singular: httproutetimeoutfilter
+  scope: Namespaced
+  versions:
+  - additionalPrinterColumns:
+    - description: The sync status of the resource with Consul
+      jsonPath: .status.conditions[?(@.type=="Synced")].status
+      name: Synced
+      type: string
+    - description: The last successful synced time of the resource with Consul
+      jsonPath: .status.lastSyncedTime
+      name: Last Synced
+      type: date
+    - description: The age of the resource
+      jsonPath: .metadata.creationTimestamp
+      name: Age
+      type: date
+    name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: HTTPRouteTimeoutFilter is the Schema for the httproutetimeoutfilters
+          API
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: HTTPRouteTimeoutFilterSpec defines the desired state of HTTPRouteTimeoutFilter
+            properties:
+              idleTimeout:
```

Comment:
Curious where the description for these guys came from. I think it makes sense to require a `min: 0` here as well. Probs worth checking with Product to see if we should implement an upper bound

Reply:
These files are all autogenerated, but I believe you can set these things with kubebuilder tags. I'll dig into that before I mark this PR as ready to review.

Reply:
You're right, today I learned about kubebuilder's [CRD validation](https://book.kubebuilder.io/reference/markers/crd-validation.html) tags 💯 
```go
// +kubebuilder:validation:Minimum
```

Reply:
It turns out you can't use the Minimum tag on time.Duration types.